src/components/footer/index.md

```diff
@@ -52,7 +52,7 @@ Make it clear whether content is available for re-use - and if it is, under what
 You can add links to:
 
 - [privacy notice](https://www.gov.uk/service-manual/design/collecting-personal-information-from-users)
-- [accessibility statement](https://www.gov.uk/service-manual/helping-people-to-use-your-service/publishing-information-about-your-services-accessibility)
+- [accessibility statement](https://www.gov.uk/government/publications/sample-accessibility-statement/sample-accessibility-statement-for-a-fictional-public-sector-website)
```

Comment:
Found a more appropriate Service Manual page for this, as this new link explains what an accessibility statement is and what it usually contains. So it covers more context than just the sample accessibility statement.

(and I'm obviously parking my thoughts on how descriptive this link is to our concurrent work happening)

```suggestion
- [accessibility statement](https://www.gov.uk/guidance/make-your-website-or-app-accessible-and-publish-an-accessibility-statement#decide-if-anything-is-a-disproportionate-burden-to-fix-right-now)
```